### PR TITLE
Update `found-modern-universal` example app with `relay-network-modern`

### PR DESCRIPTION
**Pull request by damassi**
Updates example with 

- https://github.com/relay-tools/react-relay-network-modern 
- https://github.com/relay-tools/react-relay-network-modern-ssr

And removes `fetcher` implementation.

### CODE REVIEW

**Comment by taion** [comment]
sorry, i've been slammed at work... will look at this soon

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  entry: ['babel-polyfill', './src/client'],
+
+  output: {
+    path: '/',
+    filename: 'bundle.js',
+  },
+
+  module: {
+    rules: [
+      // See: https://github.com/aws/aws-amplify/issues/686#issuecomment-387710340
```
Comment (taion) [inline index.js]: i'm surprised this is necessary... graphql.js shouldn't be getting pulled into the browser bundle, should it?

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,8 @@ pids
 *.pid
 *.seed
 
+.DS_Store
```
Comment (taion) [inline .gitignore]: can we keep this out of the gitignore here? i've generally been using a global gitignore for this sort of thing

File: package.json [hunk 3]
```diff
@@ -2,8 +2,9 @@
   "private": true,
   "scripts": {
     "lint": "eslint *.js src tools",
+    "postinstall": "cd node_modules/react-relay-network-modern-ssr && yarn build-lib",
```
Comment (taion) [inline package.json]: why is this needed btw?

**Comment by damassi** [comment]
@taion - I'm heading into a pretty intense month-long sprint right now (using `found-relay` 😄) but I'll make a note to come back to this when I have a sec

**Comment by taion** [comment]
i can probably pick this up – i would like to understand that post install hook though

**Comment by damassi** [comment]
Ok fixed those few things. Postinstall was a WIP thing until the updated version was published.

File: index.js (hunk 1 above)
Comment (damassi) [reply index.js]: This is due to `react-relay-network-modern-ssr` publishing `.mjs` files. I haven't had the time to look into this further, but this temporarily fixes it.

**Comment by taion** [comment]
Great, thanks. I can take it from here. I might drop some of the reload stuff in Express just to make things slightly simpler/leaner.

**Comment by damassi** [comment]
Alright! just removed express-reloadable, passing it over 👍

**Comment by taion** [comment]
Sorry for the delay – had a hard time getting time to update this.

**Comment by taion** [comment]
Thanks for your help in getting this example set up.
